py/htm/examples/hotgym.py

```diff
@@ -17,16 +17,21 @@
 _INPUT_FILE_PATH = os.path.join(_EXAMPLE_DIR, "gymdata.csv")
 
 default_parameters = {
- 'enc': {'resolution': 0.88, 'size': 700, 'sparsity': 0.02},
- 'sdrc_alpha': 0.1,
+  # there are 2 (3) encoders: "value" (RDSE) & "time" (DateTime weekend, timeOfDay)
+ 'enc': {
+      "value" :
+         {'resolution': 0.88, 'size': 700, 'sparsity': 0.02},
```

Comment:
so the param optimizer handles embedded/structored params (?)

Reply:
yes, this should work

Reply:
OT: but a nice addition would be specifing a suffix that optimizer does not touch. 
```
'enc' {
  'time' : { 'resolution': 0.1, 'size': 1000},
  'value_IMMUTABLE' : { <these won't be changed>},
  'other': { 'resolution: 0.01, 'size_IMMUTABLE': 1000}
}
```
I know grid_search can list ignored values, but would be nice to be able to specify that in the params. 


```